tools/graphql-schema-linter/graphql_schema_linter.test.ts

```diff
@@ -0,0 +1,9 @@
+import { makeToolTestConfig, toolTest } from "tests";
+
+toolTest({
+  toolName: "graphql-schema-linter",
```

Comment:
In theory you could add a `preCheck` hook here (or in a linter test) in order to make this work